lint-staged.config.js

```diff
@@ -46,16 +47,14 @@ module.exports = {
 
 
 /**
- * Function to remove any testing or demo files and return a string containing all file names
+ * Function to remove any testing or demo files and return an array containing all file names
  *
  * @param files
- * @returns fileNames
+ * @returns Array
  */
-function removeDirectories(files) {
-  const match = micromatch.not(files, [
-    ...IGNORE,
-    '**/testing/**',
-    '**/demo/**',
-  ]);
-  return match.join(' ');
-}
+const removeDirectories = files => micromatch.not(files, [
```

Comment:
Why were changes needed here?

Reply:
It was returning string, but I'd want it to be array, so can directly plug into the function you defined where it targets all ts and js files. And it doesn't seem it was referenced anywhere else?